live-examples/css-examples/opacity.html

```diff
@@ -0,0 +1,32 @@
+<section id="example-choice-list" class="example-choice-list large" data-property="border-radius">
+
```

Comment:
"border-radius" should be "opacity"? @schalkneethling , do we use these data attributes?

Reply:
Ineed we do. It is read and used by the `isSupported` script but, as `opacity` is so widely supported we do not need it here. Thanks for the contribution @SphinxKnight o/\o